strongbox-resources/strongbox-storage-api-resources/src/main/resources/etc/conf/strongbox.yaml

```diff
@@ -318,6 +320,10 @@ configuration:
               type: npm
               lastChangeId: 0
               replicateUrl: https://replicate.npmjs.com/
+          repositoryConfiguration:
+            type: npm
```

Comment:
We already have the `layout` for this and it's already defined as a property of the repository. Why do we need to have this here as well?